pyproject.toml

```
@@ -28,7 +28,7 @@ classifiers = [
 ]
 dependencies = [
     "CacheControl[filecache] >= 0.13.0",
-    "cyclonedx-python-lib >= 4,< 6",
+    "cyclonedx-python-lib >= 4,< 7",
```

If the API in question has been removed, then this bound needs to be changed to:

```suggestion
    "cyclonedx-python-lib >= 6,< 7",
```

---

I've changed it to >=5 since make_outputter() was added in https://github.com/CycloneDX/cyclonedx-python-lib/pull/469.

---

Thanks!